src/main/java/org/jabref/gui/MainApplication.java

```diff
@@ -0,0 +1,44 @@
+package org.jabref.gui;
+
+import java.util.List;
+
+import org.jabref.gui.FallbackExceptionHandler;
+import org.jabref.gui.openoffice.OOBibBaseConnect;
+import org.jabref.logic.importer.ParserResult;
+import org.jabref.preferences.JabRefPreferences;
+
+import javafx.application.Application;
+import javafx.stage.Stage;
+
+
+/**
+ * JabRef's main class to process command line options and to start the UI
+ */
+public class MainApplication extends Application {
```

Comment:
Could this classed be merged with JabRefGUI maybe?

Reply:
No not really. This is the main `Application` while JaberRefGui is the main `Stage`.